.github/workflows/ci.yaml

```diff
@@ -43,6 +43,16 @@ jobs:
           name: linux-build-artifacts
           path: ${{ env.BUILD_DIR }}/
 
+  lint:
```

Comment:
These are the only new changes. The rest of the changes are formatting/linting only to keep the pipelines happy.